extension/content.js

```diff
@@ -328,6 +368,11 @@ document.addEventListener('DOMContentLoaded', () => {
 			if (isCommit()) {
 				addPatchDiffLinks();
 			}
+
+			if (isPRFiles()) {
```

Comment:
It's enabled on:
- PR files listing
- PR commit pages (new)

It's not enabled on:
- commit pages (default)
- compare views